.ddev/config.yaml

```
@@ -10,29 +9,30 @@ xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
 mariadb_version: "10.2"
-nfs_mount_enabled: false
+mysql_version: ""
 provider: default
+use_dns_when_possible: true
+composer_version: ""
```

Maybe change to composer 2
```suggestion
composer_version: "2"
```

---

got adapted and tested with latest commit